docs/common/help.go

```diff
@@ -40,4 +40,28 @@ const GlobalEnvVars string = `	JFROG_CLI_LOG_LEVEL
 	CI
 		[Default: false]
 		If true, disables interactive prompts and progress bar.
+		JFROG_CLI_PLUGINS_SERVER
```

Comment:
This line (JFROG_CLI_PLUGINS_SERVER) doesn't seem to be aligned compared to "CI", "JFROG_CLI_PLUGINS_REPO" and the other env var names/